apps/trainers/models.py

```diff
@@ -27,12 +30,24 @@ def get_fullname(self):
 
 
 class Upload(models.Model):
+    def upload_filename(instance, filename):
+        filepath = f'user_{instance.trainer.user_id}/uploads/{filename}'
+        return filepath
+
     trainer = models.ForeignKey(Trainer, on_delete=models.CASCADE)
-    url = models.URLField(max_length=30)
+    title = models.CharField(max_length=100, default='Here goes text')
+    url = models.FileField(upload_to=upload_filename, null=True, blank=True, )
 
     def __str__(self):
-        return f'{self.trainer.__str__()} ({self.url})'
+        return self.trainer.__str__()
```

Comment:
Hier wäre es toll, wenn wir den title noch mit zurückgeben - macht es in der admin view etwas einfacher